pyigm/cgm/tests/test_init_cgm.py

```diff
@@ -70,3 +70,15 @@ def test_cgm_from_igmsystems():
     cgm_list = cgm_from_galaxy_igmsystems(galaxy, igmsys, chk_lowz=False)
     assert len(cgm_list) == 1
     np.testing.assert_allclose(cgm_list[0].rho.value, 127.8324005876)
+
+def test_cgm_from_igmsystems_lowz():
+    # Load sightlines 
+    sl_file = pyigm.__path__[0]+'/data/sightlines/Blind_CIV/J115120.46+543733.08.json'
+    igmsl = IGMSightline.from_json(sl_file)
+    igmsys = igmsl.make_igmsystems(vsys=400*u.km/u.s)
+    # Galaxy
+    galaxy = Galaxy((178.84787, 54.65734), z=0.00283)
+    # Go
+    cgm_list = cgm_from_galaxy_igmsystems(galaxy, igmsys, chk_lowz=True)
```

Comment:
Cool.  I've refactored to change the argument/variable name and updated the CGM class definition docstring.